procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,88 @@
+# Council Synchronous Meetings
+
+This documents procedures, norms, and logistics of the Council's sync meetings.
+
+## Frequency
+
+The Council meets once a week.
+
+After the 14th of July, 2023, the Council will re-evaluate whether this frequency still makes sense or whether it should be changed.
+
+## Location
+
+The meeting happens in some video call software (e.g., Google Meet).
+
+## Quorum
+
+The meeting has reached quorum if at *most* 2 Council representatives (and their alternates) are not present (i.e., N - 2).
+
+If quorum is not reached, then the sync meeting is canceled and the agenda is gone through asynchronously in chat.
+
+## Minutes
+
+Detailed minutes must be taken of the meeting even if the meeting was otherwise recorded. The meeting notes must be made public as soon as possible (preferably latest one week after the meeting), but before doing so Council members are asked to review those meeting notes to ensure that the notes reflect their understanding of what occurred in the meeting. Once consensus is reached that the meeting notes are accurate, the notes are released for public viewing. 
```

Comment:
I'd like to make "as soon as possible" a little more concrete, more to the level of "we promise to release minutes within X time and if not then we have not achieved one of our commitments to the community." I know this may not always be possible, but it'd be nice to be able to produce statistics about how often we meet this commitment. I see making concrete time commitments as a way of building trust with the community because over time we can point to a consistent track record of committing to things and following through.

Reply:
I tried to make this more concrete by at least requiring a public announcement when the minutes are delayed. I think that's a good first step. We'll see how this procedure works out in practice, and we can adjust from there.